cric-exchange-app-server/ormconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "default",
+  "type": "postgres",
+  "host": "localhost",
+  "port": 5432,
+  "username": "postgres",
```

Comment:
Is this an interim solution before taking it to the cloud?
I'm worried about saving these here